Resources/Prototypes/_LostParadise/Recipes/Lathes/security.yml

```
@@ -183,4 +183,78 @@
     Plastic: 2000
     Steel: 10000
-    Glass: 2500
+    Glass: 2500
+
+- type: latheRecipe
+  id: LPPMagazinePistolPF20Lethal
+  result: LPPMagazinePistolPF20lethal
+  category: Ammo
+  completetime: 5
+  materials:
+   Steel: 550
```

yamllint

[error] wrong indentation: expected 4 but found 3

(indentation)